package.json

```diff
@@ -42,5 +42,19 @@
     "singleQuote": true,
     "jsxSingleQuote": true,
     "proseWrap": "always"
+  },
+  "jest": {
+    "collectCoverageFrom": [
+      "src/**/*.{js,jsx,ts,tsx}",
+      "!<rootDir>/node_modules/"
+    ],
+    "coverageThreshold": {
```

Comment:
Are you sure of those percentages?

Reply:
Yeah, I think this "challenge" can achieve 100%